libraries/alicloud_actiontrail_trail.rb

```diff
@@ -28,10 +28,7 @@ def initialize(opts = {})
         }
       )["TrailList"]
 
-      if resp.empty?
-        @trail_name = "empty response"
-        return
-      end
```

Comment:
`@trail_name` is used to represent empty response in `to_s` method.. so it is better to leave it as it is!